packages/tests/web-test-runner.config.js

```diff
@@ -143,7 +143,8 @@ export default {
     "../lit-element/development/**/*_test.js",
   ],
   nodeResolve: true,
-  concurrency: sauceLauncher ? 1 : 10,
+  concurrency: sauceLauncher ? 1 : 10, // default cores / 2
```

Comment:
we *should* be able up the test file concurrency now as well in saucelabs, 6 seemed to work for me

Reply:
Hm, got this failure twice in a row with the production mode:

```
Using production builds
[Saucelabs] Setting up Sauce Connect proxy...

Error while running tests:
Error: 30 Sep 21:58:39 - Sauce Connect could not establish a connection.
30 Sep 21:58:39 - Please check your firewall and proxy settings.
30 Sep 21:58:39 - Goodbye.

    at Socket.<anonymous> (/home/runner/work/lit-html/lit-html/packages/tests/node_modules/saucelabs/build/index.js:316:25)
    at Socket.emit (events.js:314:20)
    at addChunk (_stream_readable.js:307:12)
    at readableAddChunk (_stream_readable.js:282:9)
    at Socket.Readable.push (_stream_readable.js:221:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:188:23)
```

But it could be totally unrelated, will retry again. Maybe some retry logic is all that's needed for this kind of failure?

Reply:
It sounds unrelated... potentially the dev + prod run follow each other too fast now? Or could it be that tunnels are still staying open on the saucelabs side? I've still been seeing this issue, even though I think we're closing the tunnel properly on our side.

Reply:
I tested the lit-next branch a lot with the iframe mode, with concurrentBrowsers set to 2 (since our open source plan doesn't allow more) and concurrency 6.

Reply:
That worked in both jobs, so yeah maybe some subtle timing thing. New times were 3m30s and 3m50s. Very nice!